worlds/kdl3/client.py

```diff
@@ -141,17 +141,18 @@ async def validate_rom(self, ctx) -> bool:
 
         ctx.game = self.game
         ctx.rom = rom_name
-        ctx.items_handling = 0b111  # always remote items
+        ctx.items_handling = 0b101  # default local items with remote start inventory
```

Comment:
Could this be a named constant instead of using a comment?

Reply:
If such a constant existed, it should be in core. Pretty sure none of the client libraries provide it as a constant either.